src/components/Header/Header.styl

```diff
@@ -1,5 +1,15 @@
 @require 'settings/breakpoints'
 
+.HeaderFixed
+    position sticky
```

Comment:
Beware, it doesn't work in IE11. Do we have officially dropped the support of IE? I remember there's a Paper about it but I can't find it.

Reply:
I think not

Reply:
Talked about it on mattermost. We don't support IE anymore so it's OK :)